File: MD_365_CRM_FRONT/src/app/Models/Product.ts
export interface Product {
    productId: string;
    productNumber: string;
    name: string;
    description: string;
    productStructure: string;
    price: string;
    price_Base: string;
    productTypeCode: string;
    quantityOnHand: string;
    standardCost: string;
    currentCost: string;
    currentCost_Base: string;
    standardCost_Base: string;
    stateCode: number;
    statusCode: number;
    stockVolume: string;
    exchangeRate: string;
    versionNumber: string;
    validFromDate: Date;
    validToDate: Date;
    createdOn: Date;
}

File: MD_365_CRM_BACK/MD_365_CRM/Models/Product.cs
namespace MD_365_CRM.Models
{
    public class Product
    {
        public Guid ProductId { get; set; }
        public string? ProductNumber { get; set; } = string.Empty;
        public string? Name { get; set; }
        public string? Description { get; set; }
        public string? ProductStructure { get; set; }
        public string? Price { get; set; }
        public string? Price_Base { get; set; }
        public string? ProductTypeCode { get; set; }
        public string? QuantityOnHand { get; set; }
        public string? StandardCost { get; set; }
        public string? CurrentCost { get; set; }
        public string? CurrentCost_Base { get; set; }
        public string? StandardCost_Base { get; set; }
        public int? StateCode { get; set; }
        public int? StatusCode { get; set; }
        public string? StockVolume { get; set; }
        public string? ExchangeRate { get; set; }
        public string? VersionNumber { get; set; }
        public DateTime? ValidFromDate { get; set; }
        public bool IsSynchronized { get; set; } = true;
        public DateTime? ValidToDate { get; set; }
        public DateTime? CreatedOn { get; set; }
    }
}
